api/src/AppBundle/v2/Fixture/Controller/FixtureController.php

```diff
@@ -197,4 +198,53 @@ public function createUser(Request $request)
 
         return $this->buildSuccessResponse($fromRequest, 'User created', Response::HTTP_OK);
     }
+
+    /**
+     * @Route("/createClientAttachDeputy", methods={"POST"})
+     * @Security("has_role('ROLE_ADMIN', 'ROLE_AD')")
+     */
+    public function createClientAndAttachToDeputy(Request $request)
+    {
+        $fromRequest = json_decode($request->getContent(), true);
+
+        $client = $this->clientFactory->create([
+            "firstName" => $fromRequest['firstName'],
+            "lastName" => $fromRequest['lastName'],
+            "phone" => $fromRequest['phone'],
+            "address" => $fromRequest['address'],
+            "address2" => $fromRequest['address2'],
+            "county" => $fromRequest['county'],
+            "postCode" => $fromRequest['postCode'],
+            "caseNumber" => $fromRequest['caseNumber'],
+        ]);
+
+        /** @var User $deputy */
+        $deputy = $this->em->getRepository(User::class)->findOneBy(['email' => $fromRequest['deputyEmail']]);
+
+        if ($deputy === null) {
+            return $this->buildNotFoundResponse(sprintf("Could not find user with email address '%s'", $fromRequest['deputyEmail']));
+        }
+
+        $deputy->addClient($client);
+
+        $this->em->persist($deputy);
+        $this->em->flush();
+
+        return $this->buildSuccessResponse($fromRequest, 'User created', Response::HTTP_OK);
```

Comment:
FWIW, `Response::HTTP_OK` is the default value so not strictly necessary
```suggestion
        return $this->buildSuccessResponse($fromRequest, 'User created', Response::HTTP_OK);
```

Reply:
Yup! I'll drop.